app/models/piece.rb

```diff
@@ -14,7 +14,7 @@ def move_to!(params)
       # Step 2: Check to see if the piece is of the oppposite color.
       if color != enemy_piece[:color]
         # Step 3: Remove piece from the board.
-        enemy_piece.update_attributes(horizontal_position: nil, vertical_position: nil)
+        enemy_piece.destroy
```

Comment:
Okay. That makes sense. Hopefully we can use the destroy method and have everything still working then.